### PR TITLE
webcam_chat fails on Windows 7 targets and Kali Linux hosts

### PR DESCRIPTION
**Pull request by todb-r7**
I couldn't get a reliable webcam_chat working on a Windows 7 target with Chrome or Firefox, from a Kali Linux attacker with Chrome or Firefox. I did manage to find some spots where I could clean things up a little (a bad variable name, more options for browser names, and 32-bit installation paths), but it still wasn't working out for me.

This PR is more of an issue than anything, since it doesn't actually solve things. I no longer stack trace, which is nice, but while I pop up both browsers, the WebRTC stuff doesn't seem to be working right on Kali -- tried with several webcams and network configurations. Maybe there's a firewall rule I'm missing somewhere on one side or the other?

Anyway, I really like this meterpreter extension. I want it to work in this common case of Kali -> Win7, since it's so fun to demo. I'd /love/ to see it work on Android Chrome, since it'd be extra creepy, but that can wait.

Assigning to @wchen-r7 aspirationally, since he wrote it initially. Someday, I'll do some more troubleshooting.

### CODE REVIEW

File: lib/rex/post/meterpreter/extensions/stdapi/webcam/webcam.rb [hunk 1]
```diff
@@ -103,7 +103,9 @@ def get_webrtc_browser_path
     case client.platform
     when /win/
       paths = [
+        "Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe",
```
Comment (Meatballs1) [inline lib/rex/post/meterpreter/extensions/stdapi/webcam/webcam.rb]: `%programfiles%`
`%programfiles(x86)%`

**Comment by hmoore-r7** [comment]
@todb-r7 Are you ok merging in @Meatballs1's recommendations (using the env path instead of fixed location)?

**Comment by bcook-r7** [comment]
Tod, could you fix this up please?

**Comment by todb-r7** [comment]
Updated in 37ac5f0, @bcook-r7 (sorry, missed the comment the first time, I've since fixed my filters).

**Comment by Meatballs1** [comment]
If you are expanding the prog files environment variables you wont want to be adding the system drive as it is already included (lines 114-115)

```
105       paths = [ 
106         "%ProgramFiles(x86)%\\Google\\Chrome\\Application\\chrome.exe", 
107         "%ProgramFiles%\\Google\\Chrome\\Application\\chrome.exe", 
108         "%ProgramW6432%\\Google\\Chrome\\Application\\chrome.exe", 
109         "%ProgramFiles(x86)%\\Mozilla Firefox\\firefox.exe", 
110         "%ProgramFiles%\\Mozilla Firefox\\firefox.exe", 
111         "%ProgramW6432%\\Mozilla Firefox\\firefox.exe" 
112       ] 
113 
 
114       drive = session.sys.config.getenv("SYSTEMDRIVE") 
115       paths = paths.map { |p| "#{drive}\\#{p}" } 
```

**Pull request merged**